docs/sources/datasources/influxdb/configure-influxdb-data-source/_index.md

```diff
@@ -60,97 +56,109 @@ Complete the following steps to set up a new InfluxDB data source:
-- **With CA cert** - Authenticate with a CA certificate. Follow the instructions of your CA (Certificate Authority) to download the certificate file.
-- **Skip TLS verify** - Toggle to bypass TLS certificate validation.
-- **Forward OAuth identity** - Forward the OAuth access token (and also the OIDC ID token if available) of the user querying the data source.
-
-**Basic auth details:**
-
-If you enable **Basic auth** under the Auth section you need to configure the following:
-
-- **User** - Add the username used to sign in to InfluxDB.
-- **Password** - Defines the token you use to query the bucket defined in **Database**. Retrieve this from the [Tokens page](https://docs.influxdata.com/influxdb/v2.0/security/tokens/view-tokens/) in the InfluxDB UI.
-
-**TLS/SSL auth details:**
+- **Product** - Select the product version of your Influx instance.
+- **Query language** - Select the query language for your InfluxDB instance. This will determine the connection details needed in **Database Settings**. The three options are:
+  - **Flux** - Flux is a data scripting language developed by InfluxData that allows you to query, analyze, and act on data. Refer to [Get started with Flux](https://docs.influxdata.com/influxdb/cloud/query-data/get-started/) for guidance on using Flux.
+  - **InfluxQL** - SQL-like language for querying InfluxDB, with statements such as SELECT, FROM, WHERE, and GROUP BY that are familiar to SQL users.
+  - **SQL** - Native SQL language starting with **InfluxDB v.3.0**. Refer to InfluxData's [SQL reference documentation](https://docs.influxdata.com/influxdb/cloud-serverless/reference/sql/) for a list of supported statements, operators, and functions.
 
-TLS/SSL certificates are encrypted and stored in the Grafana database.
+{{< admonition type="note" >}}
+_For InfluxQL only._ **Database + Retention Policy (DBRP) Mapping** must be configured before data can be queried for the following product versions: _Influx OSS 1.x_, _Influx OSS 2.x_, _Influx Enterprise 1.x_, _Influx Cloud (TSM)_, _Influx Cloud Serverless_
 
-- **CA cert** - If you toggle **With CA cert** add your self-signed cert here.
-- **Server name** - Name of the server. Example: server1.domain.com
-- **Client cert** - Add the client certificate.
-- **Client key** - Add the client key.
+Refer to [Manage DBRP Mappings](https://docs.influxdata.com/influxdb/cloud/query-data/influxql/dbrp/) for guidance on setting this up via the CLI or API 
+{{< /admonition >}}
 
-**Custom HTTP headers:**
+#### Advanced HTTP Settings (Optional)
+Advanced HTTP Settings are optional settings that can be configured for more control over your data source.
+- **Allowed cookies** - Defines which cookies are forwarded to the data source. All other cookies are deleted by default.
+- **Timeout** - Set an HTTP request timeout in seconds.
 
+##### Custom HTTP Headers
```

Comment:
Any header of this size gets a bit lost and is smaller than the text below. Could we consider sizing up headers of this size so they're more noticeable?